src/core/lib/surface/channel_init.h

```diff
@@ -33,15 +33,15 @@
 /// It also provides a universal entry path to run those mutators to build
 /// a channel stack for various subsystems.
 
-typedef struct grpc_channel_stack_builder grpc_channel_stack_builder;
-
 namespace grpc_core {
 
+class ChannelStackBuilder;
```

Comment:
Can we include channel_stack_builder.h instead of using a forward declaration here?  Or will that cause a circular dependency?

Reply:
it does yeah